### DISCUSSION TITLE
DEEPGRAM response not coming

### DISCUSSION THREAD

**Discussion by pdns90** [deepgram/community#764] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - TTS

### Details

Hello there, We are getting below error from some time, please look

error: "cURL error 28: Failed to connect to
[api.deepgram.com](http://api.deepgram.com/)
port 443: Connection timed out (see
https://curl.haxx.se/libcurl/c/libcurl-errors.html
) for
https://api.deepgram.com/v1/listen?model=general
"

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment]
@pdns90, I suspect this is a networking issue. Do you have any networking configuration or firewalls on your end? Where are you making the API requests from, and have they succeeded before?

If the request did provide a Deepgram request ID, then please share it and we can investigate and see if our logs help diagnose the issue. However, if you're not reaching Deepgram in the first place, there wouldn't be a request ID.

**Comment by pdns90** [comment]
We are not getting any request id -

Its suddenly stopped, it was working fine before yesterday.

Please see below screenshot.

error: "cURL error 28: Failed to connect to
[api.deepgram.com](http://api.deepgram.com/)
port 443: Connection timed out (see
https://curl.haxx.se/libcurl/c/libcurl-errors.html
) for
https://api.deepgram.com/v1/listen?model=general
"

**Reply by davidvonthenen** [reply]
@pdns90  Are you trying to do PreRecorded or Live transcription?

**Comment by pdns90** [comment]
Hi there, we are using PreRecorded audio file.

**Reply by davidvonthenen** [reply]
Can you ping `api.deepgram.com`?
Can you do a trace route to `api.deepgram.com`?

Please provide the entire queue string here. It's usually either an incorrect parameter or the file is not being sent.

**Comment by pdns90** [comment, marked as answer]
I am little surprise, now its automatic start again. nothing changes in our side.

Thanks,

**Comment by jkroll-deepgram** [comment]
@pdns90, glad to hear that it's working now for you, at least intermittently!

For further context, in recent days we have also been hearing similar customer reports from India and southeast Asia. Are you located in that region?

We have noticed some backbone internet issues stemming from that region of the world when trying to hit our datacenters. We don't have any direct control over that, but have filed a ticket with upstream providers.

If you do notice the issue reoccurring, can you please run this test:
1. Current time when gathering this information (UTC timestamp)
2. IP address from a machine with connectivity issues (e.g. with `curl ifconfig.me` on Linux or `Invoke-WebRequest ifconfig.me` on Windows)
3. Traceroute using `traceroute api-alt.md1.deepgram.com`

We've heard some customers say that they've gotten around the issue by using a VPN.

**Comment by pdns90** [comment]
Oh yes, I am from India and its comes without VPN as well.  did you fix anything from your side?

If we run these 3 steps, it will help to solve the issue?

Thanks

**Comment by pdns90** [comment]
Also, is there any way we can avoid this issue? as its major problem for us if deepgram API not responded.

**Comment by jkroll-deepgram** [comment]
@pdns90 the underlying issue is that our ISP, Cogent, has de-peered TATA, a major Indian ISP. I couldn't find an official announcement, but the [issue is discussed on Reddit](https://www.reddit.com/r/networking/comments/1cu13bv/cogent_depeering_tata/?rdt=53276).

Deepgram is investigating our networking options, but for an immediate-term fix, if you can use a VPN that is not backed by TATA, that will allow the networking peering to succeed.

**Comment by pdns90** [comment]
Hi Team,

As per our requirement, we needs output only in English language even user has given interview in any languages.

We are using https://api.deepgram.com/v1/listen?model=whisper&language=en&smart_format=true but response is very slow.

Could you please let me know if we needs to change any modal or any other changes required in coding.

Thanks,
PDN

**Reply by davidvonthenen** [reply]
Would recommend using `nova-2` for the model

**Reply by jkroll-deepgram** [reply]
@pdns90, note that while our Nova-2 model doesn't support translation, it offers fast and accurate transcription which can be then passed to another translation service.
